ci/jenkins/jobs/macros.yaml

```diff
@@ -69,6 +69,40 @@
 
           if !(test -f jenkins/SECRET_EXIST); then
             echo "=== Fail to get secret ==="
+            
+            echo "=== Remove too many logs ==="
```

Comment:
did you consider some simpler logic to get rid of all the log directories which are old enough? something like https://unix.stackexchange.com/a/225842

Reply:
Yes, this is a simpler logic from the aspect of time. I set up saving logs for 7 days.